src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -71,16 +74,18 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives with
                 }
               }
             } ~
-            get {
-              complete {
-                googleExtensions.createUserPetServiceAccount(WorkbenchUser(userInfo.userId, userInfo.userEmail), GoogleProject(project)).map { petSA =>
-                  StatusCodes.OK -> petSA.serviceAccount.email
+            pathEnd {
```

Comment:
this is a different, unrelated bug: the missing pathEnd directive meant that the get below and the one under pathPrefix("key") were both in play at the same time.